aws_jobs/dmarc.py

```diff
@@ -12,13 +13,12 @@
 DEFAULT_ES_RETRIEVE_SIZE = 10000
 
 
-def query_elasticsearch(session, es_region, es_url, since,
-                        es_retrieve_size=DEFAULT_ES_RETRIEVE_SIZE):
-    """Query Elasticsearch for all DMARC aggregate reports received
-    since a given time.
+def query_elasticsearch(
+    session, es_region, es_url, since, es_retrieve_size=DEFAULT_ES_RETRIEVE_SIZE
+):
+    """Query Elasticsearch for all DMARC aggregate reports received since a given time.
 
-    Parameters
-    ----------
```

Comment:
The `-----` is because I was using the [numpy docstring style](https://numpydoc.readthedocs.io/en/latest/format.html).  We don't have to use that style, but we should choose something and be consistent everywhere.

This is a question that @cisagov/team-ois should answer as a group.

Reply:
flake8 was displeased with the way the docstring was organized until I removed those. I can see if there's a way to pass flake8 but keep the numpy style if desired.

Reply:
Let's revert this one for now, and collectively decide on a docstring style going forward.  I'm not opposed to using a different style.